darshan-util/darshan-parser.c

```diff
@@ -682,25 +698,18 @@ int main(int argc, char **argv)
             printf("#\n");
             printf("# I/O timing for unique files (seconds):\n");
             printf("# ...........................\n");
-            printf("# unique files: slowest_rank_io_time: %lf\n", pdata.slowest_rank_time);
-            printf("# unique files: slowest_rank_meta_only_time: %lf\n", pdata.slowest_rank_meta_time);
+            printf("# unique files: slowest_rank_io_time: %lf\n", pdata.slowest_rank_io_total_time);
+            printf("# unique files: slowest_rank_meta_only_time: %lf\n", pdata.slowest_rank_meta_only_time);
+            printf("# unique files: slowest_rank_io_only_time: %lf\n", pdata.slowest_rank_io_only_time);
             printf("# unique files: slowest_rank: %d\n", pdata.slowest_rank_rank);
             printf("#\n");
             printf("# I/O timing for shared files (seconds):\n");
-            printf("# (multiple estimates shown; time_by_slowest is generally the most accurate)\n");
             printf("# ...........................\n");
-            printf("# shared files: time_by_cumul_io_only: %lf\n", pdata.shared_time_by_cumul);
-            printf("# shared files: time_by_cumul_meta_only: %lf\n", pdata.shared_meta_time);
-            printf("# shared files: time_by_open: %lf\n", pdata.shared_time_by_open);
-            printf("# shared files: time_by_open_lastio: %lf\n", pdata.shared_time_by_open_lastio);
-            printf("# shared files: time_by_slowest: %lf\n", pdata.shared_time_by_slowest);
+            printf("# shared files: time_by_slowest: %lf\n", pdata.shared_io_total_time_by_slowest);
             printf("#\n");
             printf("# Aggregate performance, including both shared and unique files (MiB/s):\n");
-            printf("# (multiple estimates shown; agg_perf_by_slowest is generally the most accurate)\n");
             printf("# ...........................\n");
-            printf("# agg_perf_by_cumul: %lf\n", pdata.agg_perf_by_cumul);
-            printf("# agg_perf_by_open: %lf\n", pdata.agg_perf_by_open);
-            printf("# agg_perf_by_open_lastio: %lf\n", pdata.agg_perf_by_open_lastio);
+            printf("# agg_time_by_slowest: %lf\n", pdata.agg_time_by_slowest);
```

Comment:
Note that you have a time value and a bandwidth value here, but the heading above indicates MiB/s -- may need to move the units after the value instead of in the heading? (in that case, maybe you do that for all of the output in this section)

Reply:
fixed in 5cbf2e2d..602c286e